app/pages/lab-pages-editor/context.jsx

```diff
@@ -0,0 +1,9 @@
+import { createContext, useContext } from 'react'
+
+const defaultWorkflowData = {}
+export const WorkflowContext = createContext(defaultWorkflowData)
+
+export function useWorkflowContext () {
+  const workflowData = useContext(WorkflowContext)
+  return workflowData
+}
```

Comment:
This is plain JS, not JSX, so the file should be named `context.js`.

Reply:
I'll revert this change